client/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import { Routes, Route, BrowserRouter } from "react-router-dom";
 import axios from "axios";
 
 import Landing from "./pages/Landing/Landing";
-import Loading from "./pages/Loading";
+import Loading from "./pages/Loading/Loading";
```

Comment:
Since Landing is modularized, import the module is sufficient.

```suggestion
import Loading from "./pages/Loading";
```